### PR TITLE
Extract items into own classes

### PR DESCRIPTION
**Pull request by brrritssocold**
Slaying a 10k monster.

- Extracted items into own classes
- Use strict compare (`===` and `!==`) in extracted classes
- Multi-Version items (variations, tainted / pure) **NOT** extracted

### Reasoning:
- Overview issues (Even Github web view truncates files at ~8k lines)
- Keep all item code in one place (better cohesion)

### TODO:
- Find good way to handle Multi-Version items
- Code cleanup (mostly code style)
- Create sub-packages in `Items.Consumables`, it's getting crowded in there

### CODE REVIEW

**Comment by Foxwells** [comment]
If I may ask, what does `===` offer over `==`?

**Comment by Stadler76** [comment]
http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/operators.html#strict_equality

**Comment by brrritssocold** [comment]
`==` performs type coercion, `===` does not.
[Question on SO](https://stackoverflow.com/questions/4974219/what-is-the-difference-between-and)

File: classes/classes/Items/Consumable.as [hunk 1]
```diff
@@ -31,6 +31,8 @@ package classes.Items
 		protected function get flags():DefaultDict { return kGAMECLASS.flags; }
 		protected function get camp():Camp { return kGAMECLASS.camp; }
 		protected function doNext(eventNo:Function):void { kGAMECLASS.doNext(eventNo); }
+		
+		protected var mutationsHelper:MutationsHelper = new MutationsHelper();
```
Comment (Stadler76) [inline classes/classes/Items/Consumable.as]: Thats not needed. Calling mutations.restoreLegs(...) and so on would do the trick without yet another instance.

**Pull request merged**